### PR TITLE
feat(emeis-options): add setting to force the locale of models

### PR DESCRIPTION
**Pull request by czosel**
This makes the already existing override for field locales configurable
via the emeis-options service. The existing override on model level has
been removed, because it's not accessible when ember-emeis is used as an
engine.

The override is useful for cases when translations for specific models
are not required / wanted.

### CODE REVIEW

**Review by winged — APPROVED** [review]
LGTM!

File: addon/templates/scopes/edit.hbs [hunk 1]
```diff
@@ -6,12 +6,12 @@
         @updateModel={{this.updateModel}}
         @listViewRouteName="scopes.index"
       >
-        <EditForm::Element @label={{t "emeis.scopes.headings.name"}}>
+        <EditForm::Element @label={{t "emeis.scopes.headings.fullName"}}>
```
Comment (czosel) [inline addon/templates/scopes/edit.hbs:9]: drive-by fix
